tests/src/Traits/ConfigurationBackupTrait.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/*
+ * Copyright the Collabora Online contributors.
```

Comment:
We cannot claim copyright if we copied from elsewhere..

(in fact the authors happen to be mostly the same people. The original package openeuropa/oe_showcase has no explicit copyright notice, but the copyright holder can be assumed how it is published)